starter-code/css/main.css

```diff
@@ -133,6 +152,7 @@ h5 {
 .bar-white {
     width: 450px;
     border: 2px solid #FFF;
+    background-color: #fff;
```

Comment:
Hey Corrado,

This is perfect! Nothing to say except I love the logo animation 😉